pages/services/hdfs/2.5.0-2.6.0-cdh5.11.0/release-notes/index.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+excerpt: Release notes for v. 2.5.0-2.6.0-cdh5.11.0
+title: Release Notes
+menuWeight: 10
+model: /services/hdfs/data.yml
+render: mustache
+---
+## Version 2.5.0-2.6.0-cdh5.11.0
+### Feature
+- Hadoop `ipc.maximum.data.length` property is now available in service configuration.
+- Readiness-check interval, timeout and delay are now available as configurable properties.
+- RLIMIT is now a configurable property.
+
+### Bug Fixes
+- Fix for JournalNodes failing to establish quorum when paxos folder is missing.
```

Comment:
Quoting `'paxos'` is probably a good idea

Reply:
Updated.